io.catenax.shared.business_partner_certificate/RELEASE_NOTES.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to this model will be documented in this file.
+
+## [1.0.0] 04.03.2024
```

Comment:
change the date to 11th March.

Reply:
> change the date to 11th March.

Hi @agg3fe, I have adapted date to 11th March.